webpack/assets/javascripts/channels/redux_channel.js

```diff
@@ -0,0 +1,16 @@
+import store from '../react_app/redux';
+import consumer from './consumer';
+
+consumer.subscriptions.create('ReduxChannel', {
+  connected() {
+    // Called when the subscription is ready for use on the server
+  },
+
+  disconnected() {
+    // Called when the subscription has been terminated by the server
+  },
+
+  received({ action }) {
+    store.dispatch(action);
```

Comment:
IMHO this approach is somewhat limited and confusing, the server responds only with actions
instead, a client action can just subscribe to a channel, and store its data in a unique key in our store, just like our API ( a middleware can fit for this).